Stock/StockHooks.cs

```diff
@@ -0,0 +1,154 @@
+﻿using Mono.Cecil.Cil;
+using MonoMod.Cil;
+using System;
+using Terraria;
+using Terraria.ModLoader;
+using Terraria.UI;
+using Terraria.UI.Chat;
+
+namespace StockableShops.Stock;
+
+internal class StockHooks : ModSystem
+{
+    #region decrease the stocked stack when buying an item
```

Comment:
I don't use regions, and I'd prefer these are removed for consistency within the mod's source.